rpc_jobs/rpc_upgrades.yml

```diff
@@ -0,0 +1,96 @@
+### tests rpc-upgrades to <scenario> leaped from <action>
+- project:
+    name: "rpc-upgrades-aio-newton-head"
+    repo_name: "rpc-upgrades"
+    repo_url: "http://github.com/rcbops/rpc-upgrades"
+    series: "newton"
+    branches:
+     - "master"
+    image:
+      - aio:
+          FLAVOR: "performance2-15"
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "newton"
+    action:
+      - "liberty"
+#      - "r12.2.8"
+#      - "r12.2.5"
+#      - "r12.2.2"
+#      - "r12.1.2"
+#      - "kilo" 
+    jira_project_key: "RLM"
+    jobs:
+      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
+      - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
```

Comment:
I'm not a fan of how this naming is being done. For example:
This will create the job PR_rpc-upgrades-newton-aio-newton-liberty which says that a liberty to newton leapfrog upgrade will be done. No one can tell from that job name what the build is upgrading from/to. It's not clear it's a leapfrog upgrade. The use of `"aio"` to represent the image is inconsistent with other jobs. This is being tested using the master branch of rpc-upgrades whereas someone familiar with other job names would think this was the newton branch.
Contrast that with this existing job, PR_rpc-openstack-newton-xenial-swift-deploy, from which I can see a Swift deploy on Xenial is being done using the newton branch of rpc-openstack.

I'd rather see the standard template adjusted or have rpc-upgrades compress additional data into one of these elements than go with this format.

Reply:
Yeah, I'm definitely open to adjusting the template, with series I could make things unique for the destination (newton-aio, newton-mnaio, r14.4.1-aio, newton-mnaio), use scenario to notate leap, and action to notate original deployed version.

The problem comes in with the Post Merge jobs and the fact we're only using a master branch.  The way the existing templates are, this makes it hard to make unique jobs for all the projects/versions I need to build.  

Originally how I wanted it ( tests rpc-upgrades to <series> leaped from <action> ):

    PR_{repo_name}-{series}-{image}-{scenario}-{action}
    PM_{repo_name}-{branch}-{image}-{scenario}-{action}

    #### rpc-upgrades-newton-aio (series newton_aio)
    PR_rpc-upgrades-newton_aio-trusty-leap-liberty
    PR_rpc-upgrades-newton_aio-trusty-leap-r12.2.8
    PM_rpc-upgrades-master-trusty-leap-liberty
    PM_rpc-upgrades-master-trusty-leap-r12.2.8

    #### rpc-upgrades-r14.5.0-aio (series r14.5.0_aio)
    PR_rpc-upgrades-r14.5.0_aio-trusty-leap-liberty
    PR_rpc-upgrades-r14.5.0_aio-trusty-leap-r12.2.8

    PM_rpc-upgrades-master-trusty-leap-liberty    ############ Duplicate
    PM_rpc-upgrades-master-trusty-leap-r12.2.8    ############ Duplicate

This makes it hard to make the jobs unique, hence the odd naming the jobs get generated as.  I'm totally open to suggestions and fully acknowledge this will probably change but would at least like to start testing the job automation to move forward and am willing to rename to make them make more sense.